Xamarin.Essentials/Barometer/Barometer.tizen.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Tizen.Sensor;
+using TizenBarometerSensor = Tizen.Sensor.PressureSensor;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Barometer
+    {
+        static TizenBarometerSensor DefaultSensor
+            => (TizenBarometerSensor)Platform.GetDefaultSensor(SensorType.Barometer);
+
+        internal static bool IsSupported
+            => TizenBarometerSensor.IsSupported;
+
+        internal static void PlatformStart(SensorSpeed sensorSpeed)
+        {
+            DefaultSensor.Interval = sensorSpeed.ToPlatform();
+            DefaultSensor.DataUpdated += DataUpdated;
+            DefaultSensor.Start();
+        }
+
+        internal static void PlatformStop()
+        {
+            DefaultSensor.DataUpdated -= DataUpdated;
+            DefaultSensor.Stop();
+        }
+
+        static void DataUpdated(object sender, PressureSensorDataUpdatedEventArgs e)
+        {
+            OnChanged(new BarometerData(e.Pressure));
```

Comment:
What is your unit of pressure?  We always return `hectopascals`, want to make sure yours is the same.

Reply:
Yes, It is `hPa (hectopascals)`. 
https://developer.tizen.org/ko/development/guides/.net-application/location-and-sensors/device-sensors#pressure